src/Providers/RouterProvider.php

```diff
@@ -20,14 +21,16 @@ class RouterProvider extends AbstractServiceProvider
      */
     public function register()
     {
-        $this->container->share(
+        $this->getContainer()->share(
```

Comment:
What's the benefit of using `getContainer` over `container`?

Reply:
Nothing really, but we should pick one and use it consistently.

:+1:  for ->container
:-1:  for ->getContainer()